docs/vscode-extension/migrate-v1/migrate-v1-tab.md

```diff
@@ -0,0 +1,47 @@
+# Build UI-based Apps for Teams
+Microsoft Teams supports the ability to run web-based UI inside "custom tabs" that users can install either for just themselves (personal tabs) or within a team or group chat context.
+
+## Prerequisites
+- [NodeJS](https://nodejs.org/en/)
+- An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) 
+
+##  Initialize V1 tab project with the latest Teams Toolkit
```

Comment:
Move the migration command to the parent doc.

Different user scenarios require different docs:
- Users who are going to migrate
  1. Go to the online "migrate-v1" doc and see how to migrate
  2. Go to the online "migrate-v1-tab" doc and understand followups after migration
  3. Do migration
- Users who already migrated
  1. Go to the local "migrate-v1" doc and see what happened
  2. Go to the local "migrate-v1" doc and see what should do next
  3. Go to the online doc if want to read more